src/main/ruby/core/rbconfig.rb

```diff
@@ -170,7 +170,6 @@ module RbConfig
                     '-Wno-unknown-warning-option',            # If we're on an earlier version of clang without a warning option, ignore it
                     '-Wno-int-conversion',                    # MRI has VALUE defined as long while we have it as void*
                     '-Wno-int-to-pointer-cast',               # Same as above
-                    '-Wno-macro-redefined',                   # We redefine __DARWIN_ALIAS_C
```

Comment:
Do we not redefine `__DARWIN_ALIAS_C` any more?

Reply:
It seems not, I cannot find it in the source base. It was removed in f4aa6645775db98923a0189fd16bdf75c3bf8ee4 and doesn't seem needed anymore.